ml_git/ml_git_message.py

```diff
@@ -162,7 +161,8 @@
     'INFO_INITIALIZING_ENTITY_DOWNLOAD': 'Initializing related %s download',
     'INFO_ALREADY_TAG': 'already at tag [%s]',
     'INFO_SPEC_NOT_HAVE_MUTABILITY': 'The spec does not have the \'mutability\' property set. Default: strict.',
-    'INFO_STARTING_INTEGRITY_CHECK': 'Starting integrity check on [%s]',
+    'INFO_STARTING_INTEGRITY_CHECK': 'Starting integrity check on [{}]',
+    'INFO_FINISH_INTEGRITY_CHECK': 'Finish integrity check on [{}]',
```

Comment:
```suggestion
    'INFO_FINISH_INTEGRITY_CHECK': 'Finished integrity check on [{}]',
```
Or is it asking the user to finish an integrity check?